s3/libirods_s3.cpp

```diff
@@ -848,20 +849,22 @@ static void mrdWorkerThread (
             rangeData.pCtx = &bucketContext;
             rangeData.prop_map_ptr = &_prop_map;
 
-            msg.str( std::string() ); // Clear
-            msg << "Multirange:  Start range " << (int)seq << ", key \"" << g_mrdKey << "\", offset "
-                << (long)rangeData.get_object_data.offset << ", len " << (int)rangeData.get_object_data.contentLength;
-            rodsLog( LOG_DEBUG, msg.str().c_str() );
+            irods::log(LOG_DEBUG,
+                    fmt::format("Multirange:  Start range {}  \"{}\", offset {}, len {}",
+                    (int)seq,
+                    g_mrdKey,
+                    (int64_t)rangeData.get_object_data.offset,
+                    (int)rangeData.get_object_data.contentLength));
```

Comment:
Are the C-style casts necessary here?

fmtlib deduces the correct types for you.

Reply:
I don't think so and I was sure you were going to ask.  ;-)

Reply:
Done both here and in transport code.